rockcraft-pack/action.yml

```diff
@@ -14,6 +14,12 @@ inputs:
 
       The default is 'trace'.
     default: 'trace'
+  rockcraft-channel:
+    description: >
+      Install the snap from a specific channel
+
+      If not provided, it defaults to stable.
+    default: ''
```

Comment:
Thanks @addyess 

At first glance, it lgtm. The `default` should be `stable` though

Reply:
i don't mind updating the default.  

i saw in [rockcraft-pack-action.ts](https://github.com/canonical/craft-actions/blob/main/src/rockcraft-pack-action.ts#L11) that if the value was empty, `stable` would get picked up

Reply:
yes true, but it's a slightly different default. In the code we're saying "if we don't know what the channel is, then default to `latest/default`". That assumption (i.e. `"" -> 'stable'`) could change. 

In this case, we're providing a user-facing default for what we believe the action should run as. This way, tomorrow the code can change but the action will still behave the same.

Reply:
👍🏻